### DISCUSSION TITLE
Making the documentation window enter-able to allow interaction with the docs (especially links)

### DISCUSSION THREAD

**Discussion by ghost** [hrsh7th/nvim-cmp#1970] [Ideas]
When selecting a completion suggestion, a documentation popup menu opens up which is scroll-able but not ente-rable.... It would be great if the documentation window was actually enter-able which would allow copying the docs and entering the links... this is currently not possible with window commands as the window's `enter` parameter is set to false as pointed out by @Shougo in this Q/A [discussion/1969](https://github.com/hrsh7th/nvim-cmp/discussions/1969)

However a previous [discussion/1517](https://github.com/hrsh7th/nvim-cmp/discussions/1517) did report success in achieving this... I could not replicate the solution

**Comment by Shougo** [comment]
I think the documentation buffer number can be get from user, it can possible.

**Reply by ghost** [reply]
> I think the documentation buffer number can be get from user, it can possible.

That's great! Can it be done from outside the plugin codebase? Any api functions or something?

**Reply by Shougo** [reply]
Unfortunately, it is not exposed.

```lua
    vim.api.nvim_buf_call(self.window:get_buffer(), function()
      vim.cmd([[syntax clear]])
      vim.api.nvim_buf_set_lines(self.window:get_buffer(), 0, -1, false, {})
    end)
```

`self.window:get_buffer()` returns buffer number.
And the buffer name is auto generated.
The new API is needed.

**Reply by ghost** [reply]
@Shougo 
Oh cool... maybe I can look into it sometime... doesnt seem too complicated to implement... then again...
